packages/core/utils/src/modules-sdk/definition.ts

```diff
@@ -24,6 +24,7 @@ export const Modules = {
   FILE: "File",
   NOTIFICATION: "Notification",
   INDEX: "Index",
+  LOCKING: "Locking",
```

Comment:
todo: Should be `locking`
```suggestion
  LOCKING: "locking",
```